src/api/java/com/direwolf20/buildinggadgets/api/materials/UniqueItem.java

```diff
@@ -43,6 +44,10 @@ public Item getItem() {
         return item;
     }
 
+    public ItemStack toItemStack() {
+        return new ItemStack(item, 1, tagCompound);
```

Comment:
This needs a Factory to be passed into the UniqueItem, which may default to this. An UniqueItem is not required to have all the nbt data, therefore this might throw an exception if it no longer matches the stacks Format.

Reply:
What do you mean by a factory?
If it will go into the parameter, we will have a deserialization problem without adding yet another registry. I think we can just give it an empty tag if the UniqueItem doesn't have complete data -- ItemStack's created will have no tag by default, and all mods must handle that properly by default.

Reply:
You're correct - I forgot that we need to sync that to the client... Well then this isn't going to happen soon